kogito-scripts-quarkus/README.md

```diff
@@ -7,10 +7,21 @@ A quickstart project is the simplest hello world kind of example, it accepts inp
 This example shows
 
 * invoking scripts from within process
-	
-	
+		
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
 
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
```

Comment:
I find it hard to understand correctly what is this second Daigram Properties. Took me some time to understand that we were on the same properties

Maybe add `(top)` and `(bottom)` to distinguish ?